### DISCUSSION TITLE
Restore the product comparison engine

### DISCUSSION THREAD

**Discussion by ghost** [PrestaShop/PrestaShop#40187] [Ideas]
Hi.
Comparison engine was useful in some use cases. Please consider to bring that functionality back in future releases.

Thanks!

**PS> If You want to bring that function back please add thumb up to show PrestaShop team that case matters.**

**Comment by khouloudbelguith** [comment]
Hi @PrestaShark,

Thanks for your report.
We removed this native module in 1.7.
http://forge.prestashop.com/browse/BOOM-4092
@marionf, @colinegin what do you think? is it possible to restore this function in the next major version PrestaShop?

Thanks!

**Comment by MatShir** [comment]
Hi @PrestaShark,
Your feedback has been shared with the team. But as you said it's a feature really useful only for some cases. There are many modules that can do the job in the marketplace for these specific needs.

However, I post your feature request on [http://feedback.prestashop.com](http://feedback.prestashop.com/forums/387864-prestashop-1-7-x), it will help us to prioritize all the idea from the community and bring more involvement around the feature. 

Thank you for your implication,

**Comment by ghost** [comment]
Make comparison engine great again ;)

https://netronix.pl/en/products-comparison?produkt=24|79
